app4java/src/main/java/com/nyxei/pollingus/PollingusApplication.java

```diff
@@ -1,13 +1,37 @@
 package com.nyxei.pollingus;
 
-import org.springframework.boot.SpringApplication;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
 public class PollingusApplication {
 
 	public static void main(String[] args) {
-		SpringApplication.run(PollingusApplication.class, args);
+		 
+
+		try {
+			HttpRequest request = HttpRequest.newBuilder()
+				.uri(new URI("https://graph.facebook.com/v15.0/102062986090216/messages"))
+				.header("Authorization", "Bearer EAAGYN0OOscYBAKr9XGGCllZBFZC9XAIJDwAVGAlkDMWdhMElo6aJbWbsMu2W7pZCfObN39uAHqdZAcDNx5qmPlVv88wmiZBva7YnEqzJ3C2hzUiZAFZCdD9zCjUQ193touuz6qqE3VNgwCKtYhW0Cq8bkgxtgdx2COdx5VRwsEcbQjO3w7VQazLgMvy7TxsR9ZANIvIgdFcYTWY1RZAvwDuSH")
```

Comment:
Same goes for all the params you use.

Reply:
final String Token = "Bearer EAAGYN0OOscYBAJAlwOwszT65lOTGZAFG29ZA3A9DBJZADGEdOCNZChoEm9eE34pWSo6WYWuDOs2pBKCCNGoiqqhBp9uRnjFpU61ZAqlxalJXGJFfcIDZADATBCZBhe2MZATNJzFkjCVZAXsYJ3vwrdjOZCDuNus7ZA6390ZASSK9bJWY04lU0RJpKXeYz2Bd48G232kMy1bzM8Be9QZDZD";
		final String App_json = "application/json"